modular_nova/modules/tarkon/code/misc_fluff/research.dm

```diff
@@ -91,9 +91,12 @@
 	return ..()
 
 /obj/machinery/rnd/server/tarkon/add_context(atom/source, list/context, obj/item/held_item, mob/user)
-	if(held_item && istype(held_item, /obj/item/research_notes))
```

Comment:
Hi just wanted to drop by.

There is no need to touch this proc. Simply remove this line

https://github.com/Fluffy-Frontier/FluffySTG/blob/677c72ce6bc219e502adda49d761c3353db88641/modular_nova/modules/tarkon/code/misc_fluff/research.dm#L88

Because `/obj/machinery/rnd` already does this so for all your custom subtypes just remove `register_context()`

Reply:
Actually you should call the parent proc as well for `add_context()` so they can inherit those tooltips. Do that too

Reply:
:eyes:

Thanks! I haven't quite understand when this "overriden when it shouldn't" happen. Also, if you look at colony fabricator, (the part I commented out since its only for test merging) - you could understand, why GoldenAlpharex was worried about removing `user_try_print_id` proc